src/workspace-element.coffee

```diff
@@ -95,6 +95,50 @@ class WorkspaceElement extends HTMLElement
 
   focusPaneViewOnRight: -> @paneContainer.focusPaneViewOnRight()
 
+  copyItemToPaneOnLeft: (destroyOriginal) ->
```

Comment:
I'm a little confused about this `destroyOriginal` flag. Seems like if we wanted to destroy the original, maybe we should just *move* the item. Maybe another set of methods, `moveItemToPaneOn*` would be clearer and more efficient in this case.